lib/collection/src/shards/replica_set.rs

```diff
@@ -1170,7 +1170,9 @@ impl ShardReplicaSet {
     ) -> CollectionResult<Option<PointIdType>> {
         let read_local = self.local.read().await;
         if let Some(ForwardProxy(proxy)) = &*read_local {
-            proxy.transfer_batch(offset, batch_size).await
+            proxy
+                .transfer_batch(offset, batch_size, &self.update_runtime)
```

Comment:
The replica only has the `update_runtime` to be used to drive the underlying scroll request for the batch transfer.